src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsKernel.hpp

```diff
@@ -194,6 +194,8 @@ class Multiphase :
       localDispFlowJacobian{ {0.0} },
       localFlowDispJacobian{ {0.0} },
       localFlowFlowJacobian{ {0.0} },
+      localVolBalanceResidual( 0.0 ),
+      localVolBalanceJacobian{ 0 },
```

Comment:
```suggestion
      localVolBalanceResidual{ 0 },
      localVolBalanceJacobian{ {0.0} },
```

For consistency with what is done in the single-phase poromechanics kernel (see also suggestions that that follow).